docs/contributing.rst

```diff
@@ -132,6 +132,29 @@ mean that this work should be relatively manageable.
 We hope that the lessons we learn building something useful in the short term will be
 applicable in the future as other needs arise.
 
+.. _backwards compatibility with meica:
```

Comment:
Can you add this section to the TOC ? Also, I think it might make sense to place this before "how does tedana future-proof its development" since that directly feeds into the release checklist !

Reply:
Done ✅